Quantum-Control-Applications/Superconducting/Single Fixed Transmon/DRAG_calibration.py

```diff
@@ -0,0 +1,136 @@
+"""
+An experiment to calibrate the DRAG coefficient
+"""
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+import numpy as np
+from qm import SimulationConfig
+
+###################
+# The QUA program #
+###################
+
+n_avg = 1000
+
+cooldown_time = 5 * qubit_T1 // 4
+
+a_min = -0.1
+a_max = 0.3
+da = 0.005
+amps = np.arange(a_min, a_max + da / 2, da)  # + da/2 to add a_max to amplitudes
+
+with program() as drag:
+    n = declare(int)
+    n_st = declare_stream()
+    a = declare(fixed)
+    I = declare(fixed)
+    Q = declare(fixed)
+    I_st = declare_stream()
+    Q_st = declare_stream()
+    state = declare(bool)
+    state_st = declare_stream()
```

Comment:
Not very important but I would prefer to call it state1 to be consistent with state2, what do you think?

Reply:
sounds cool